apps/Pycharm CE/install

```diff
@@ -2,7 +2,7 @@
 
 install_packages python3-pip python3-dev openjdk-11-jdk build-essential clang || exit 1
 
-wget https://download.jetbrains.com/python/pycharm-community-2021.3.tar.gz -O pycharm-community.tar.gz || error "Failed to download pycharm-community.tar.gz"
+wget https://download.jetbrains.com/python/pycharm-community-2021.3.1.tar.gz -O pycharm-community.tar.gz || error "Failed to download pycharm-community.tar.gz"
```

Comment:
Use `version` to prepare for [future GitHub actions update script](https://github.com/Botspot/pi-apps/pull/1330).

Also, the `wget` function already prompts users for errors, so we don't need to prompt again.

```suggestion
version="2021.3.1"

wget https://download.jetbrains.com/python/pycharm-community-${version}.tar.gz -O pycharm-community.tar.gz || exit 1
```